controllers/disruption_rollout_controller.go

```diff
@@ -32,11 +37,25 @@ func (r *DisruptionRolloutReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	instance := &chaosv1beta1.DisruptionRollout{}
 	randSource := rand.New(rand.NewSource(time.Now().UnixNano()))
 
+	// reconcile metrics
+	r.handleMetricSinkError(r.MetricsSink.MetricReconcile([]string{"controller", r.MetricsSink.GetSinkName()}))
+
+	defer func(tsStart time.Time) {
+		tags := []string{}
+		if instance.Name != "" {
+			tags = append(tags, "rolloutName:"+instance.Name, "namespace:"+instance.Namespace)
```

Comment:
i know it's wordier, but why not be consistent with the `name` tags for the other two controllers: `disruptionRolloutName`?

Reply:
nvm, i see you do so below, this one must just have been forgotten to update

Reply:
Yeah, I briefly forgot the prefix gives a ton of this context so there is no reason to differentiate in the tags.

Reply:
I will leave it for the logs considering those arent prefixed with any context.

Reply:
sorry, i did a bad job explaining what i meant, I should have written `*name`. See my latest comment, but I think we should be consistent with the disruption_controller's existing metric tags